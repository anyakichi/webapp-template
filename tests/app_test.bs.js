// Generated by ReScript, PLEASE EDIT WITH CARE

import * as Jest from "@glennsl/rescript-jest/src/jest.bs.js";
import * as React from "react";
import * as App$WebappTemplate from "../src/App.bs.js";
import * as ReactTestingLibrary from "rescript-react-testing-library/src/ReactTestingLibrary.bs.js";

function toMatchSnapshot(a) {
  return Jest.Expect.toMatchSnapshot(a);
}

Jest.describe("App", (function (param) {
        beforeEach(function () {
              return ReactTestingLibrary.renderOnScreen(React.createElement(App$WebappTemplate.make, {}));
            });
        return Jest.test("test title", (function (param) {
                      return Jest.Expect.toMatchSnapshot(Jest.Expect.expect(ReactTestingLibrary.getByRole(undefined, {
                                          NAME: "Str",
                                          VAL: "heading"
                                        }, document)));
                    }));
      }));

export {
  toMatchSnapshot ,
  
}
/*  Not a pure module */
