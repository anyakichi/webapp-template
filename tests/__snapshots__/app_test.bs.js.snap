// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test title 1`] = `
<h1
  class="text-center p-3"
>
  Web Application Template
</h1>
`;
